erpnext/stock/doctype/item_alternative/item_alternative.py

```diff
@@ -43,13 +43,26 @@ def validate_duplicate(self):
 			frappe.throw(_("Already record exists for the item {0}".format(self.item_code)))
 
 def get_alternative_items(doctype, txt, searchfield, start, page_len, filters):
-	return frappe.db.sql(""" (select alternative_item_code from `tabItem Alternative`
-			where item_code = %(item_code)s and alternative_item_code like %(txt)s)
+	return frappe.db.sql("""
+		(select
+			alternative_item_code, alternative_item_name
```

Comment:
- These fields should be dynamically generated based on title field
- check queries.py for reference